content/blog/cdf-intro-uber-crane/index.md

```diff
@@ -19,33 +19,34 @@ A few surprising facts from the post:
 
 There are in-depth descriptions of how Uber tackles those challenges throughout the post, which makes an excellent read.
 
-We thought it would be a fun thought experiment to run through how OpenFABR CDF could help resolve some of the challenges.
+We thought it would be a fun thought experiment to run through how FABR could help resolve some of the challenges.
 
 ## Repeatable infrastructure & Layered Dependencies
 
 The *Zone Turn-Up* section of the post describes two challenges:
 
-- Manual infrastructure management - made it slow to spin up a new zone. They adopted Infrastructure-as-Code (IaC) to solve this, nothing ground breaking.
+- Manual infrastructure management - made it slow to spin up a new zone. They adopted Infrastructure-as-Code (IaC) to solve it which is a natural choice.
```

Comment:
Think it should be more positive in the tone of language when we talk about their approaches.